### PR TITLE
[WIP] Product page updates

### PR DESCRIPTION
**Pull request by Dantheman720**
This is a master branch that incorporates product page changes for Summit/DevNation.

### CODE REVIEW

**Comment by Dantheman720** [comment]
Commit 22f8a21139704ffdfe89cabf382510ec55f8ec89 incorporates DEVELOPER-3156.

**Comment by Dantheman720** [comment]
@insectengine @adelasofia - I'm using this as the placeholder for product changes. Is there already a branch open for DevNation/Summit changes?

**Comment by adelasofia** [comment]
@Dantheman720 I don't think there is. Just my Stack Overflow branch which will go live before summit too, but we probably want to keep them separate

**Comment by Dantheman720** [comment]
@lincolnthree - This is what I'm using as a placeholder for product updates. Just in case you were wondering where my updates where going.

**Comment by paulrobinson** [comment]
retest this please

**Comment by Dantheman720** [comment]
@paulrobinson - The acceptance tests will fail for now since the tests haven't been updated for the primary nav.

**Comment by Dantheman720** [comment]
retest this please.

**Comment by Dantheman720** [comment]
retest this please.

**Comment by Dantheman720** [comment]
retest this please.

**Comment by Dantheman720** [comment]
retest this please.

**Comment by adelasofia** [comment]
@RubyGemTSL any idea why the tests don't finish running? They have stayed in 'pending' in the past builds in this PR too.

File: products/cdk/get-started.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ Build your first container
 ## Step1 Windows Content
 The easiest way to install the Red Hat Container Development Kit is to use the Red Hat Development Suite installer.
 
-. Download the link:http://insert-link-here[Red Hat Development Suite 1.0] installer
+. Download the link:https://developers.redhat.com/download-manager/file/development-suite-1.0.0-GA-bundle-installer.exe[Red Hat Development Suite 1.0] installer
```
Comment (pmuir) [inline products/cdk/get-started.adoc]: This link actually needs to be https://developers.redhat.com/download-manager/file/cdk/development-suite-1.0.0-GA-bundle-installer.exe to get it to redirect to the right get-started page (this one) - can we get that fixed.

**Comment by pmuir** [comment]
The CDK get started instructions are still not displaying properly.

**Comment by paulrobinson** [comment]
Replaced

**Pull request closed without merging**